### ISSUE TITLE
Not compilable on M1 Mac

### ISSUE THREAD

**Issue by StephanBeutel** [impsnldavid/imp.push#10]
Is there a plan to upgrade the code to work on M1 Macs?
I'm not able to compile it because of outdated project configuration and platform

**Comment by kirillsunrise** [comment]
I agree, it would be great if David could modify it to support M1 Macs.

**Comment by tonydavidcray** [comment]
Adding in a voice of praise for this fantastic project - have created numerous projects with it. Would love to see an ARM version also, but completely understand the work required.
